source/main.cc

```diff
@@ -375,6 +377,9 @@ int main (int argc, char *argv[])
   // as deemed useful by TBB
   Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv, /*n_threads =*/ 1);
 
+  // enable floating point exceptions
+  feenableexcept(FE_DIVBYZERO|FE_INVALID);
```

Comment:
I mean, is it expensive to run with this enabled? We could do it only if debug mode is enabled.